lib/config.js

```diff
@@ -32,16 +32,16 @@ function defaultConfigPaths () {
 function getInCluster () {
   const host = process.env.KUBERNETES_SERVICE_HOST
   const port = process.env.KUBERNETES_SERVICE_PORT
-
+  const root = process.env.RELATIVE_ROOT || '/'
```

Comment:
Can we pick a kubernetes-client specific name for this environment variable?  How about `KUBERNETES_CLIENT_SERVICEACCOUNT_ROOT`?

Reply:
Sounds good, will do asap